frontend/src/metabase/ui/index.ts

```diff
@@ -1,8 +1 @@
-export { ThemeProvider } from "metabase/ui/components/ThemeProvider";
-export { theme } from "metabase/ui/theme";
-
-// General helpers
-export { Box, Group } from "@mantine/core";
```

Comment:
@ranquild what do you think about just exporting from the index file when we don't need to do any tweaking? I don't feel like it's necessary to create a chain of re-exports when we're not modifying them at all, and if we do want to actually tweak them later, it should be easy to do as long as we always import from `metabase/ui`

Reply:
I just thought that we'd likely want to add stories for them like we do for radio, and I liked that `Radio` component is exported near the story file.

Reply:
Yeah I think it provides a clearer path to adding additional documentation this way so while it seems a little extra right now it might be worth it in the long run.